src/main/java/subway/line/LineService.java

```diff
@@ -24,35 +25,55 @@ public LineService(LineRepository lineRepository, StationRepository stationRepos
     @Transactional
     public LineResponse saveLine(LineRequest lineRequest) {
         Station upStation = stationRepository.findById(lineRequest.getUpStationId())
-                .orElseThrow(() -> new EntityNotFoundException());
+                .orElseThrow(EntityNotFoundException::new);
         Station downStation = stationRepository.findById(lineRequest.getDownStationId())
-                .orElseThrow(() -> new EntityNotFoundException());
+                .orElseThrow(EntityNotFoundException::new);
 
-        Line line = lineRepository.save(new Line(lineRequest.getName(), lineRequest.getColor(), upStation, downStation));
-        return line.createLineResponse();
+        Line line = lineRepository.save(new Line(lineRequest.getName(), lineRequest.getColor(), upStation, downStation, lineRequest.getDistance()));
+        return new LineResponse(line);
     }
 
     public List<LineResponse> findAllLines() {
         return lineRepository.findAll().stream()
-                .map(Line::createLineResponse)
+                .map(LineResponse::new)
                 .collect(Collectors.toList());
     }
 
     public LineResponse findLineById(Long id) {
-        return lineRepository.findById(id)
-                .orElseThrow(() -> new EntityNotFoundException())
-                .createLineResponse();
+        Line line = lineRepository.findById(id)
+                .orElseThrow(EntityNotFoundException::new);
+        return new LineResponse(line);
     }
 
     @Transactional
     public void updateLineById(Long id, LineUpdateRequest lineUpdateRequest) {
         lineRepository.findById(id)
-                .orElseThrow(() -> new EntityNotFoundException())
-                .update(lineUpdateRequest);
+                .orElseThrow(EntityNotFoundException::new)
+                .update(lineUpdateRequest.getName(), lineUpdateRequest.getColor());
```

Comment:
이부분은 오히려 분리하는 게 가독성이 좋다고 생각됩니다 😄 
다른 메서드들에서는 분리하시기도 했구요. 일관성을 가져야 코드를 읽기 편해집니다 👍 

```suggestion
                .orElseThrow(EntityNotFoundException::new);
                
                line.update(lineUpdateRequest.getName(), lineUpdateRequest.getColor());
```